backend/src-common/src/main/java/com/siemens/sw360/projects/Sw360ThriftServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.siemens.sw360.projects;
+
+import org.apache.log4j.Logger;
+import org.apache.thrift.TProcessor;
+import org.apache.thrift.protocol.TProtocolFactory;
+import org.apache.thrift.server.TServlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.apache.log4j.Logger.getLogger;
+
+/**
+ * Created by andreas on 27.11.15.
+ */
+public class Sw360ThriftServlet extends TServlet {
```

Comment:
Shouldn't that be `Sw360ThriftServlet`? I thougt one of the goals of this PR was, to replace all `TServlet`'s